src/cmd/run.go

```diff
@@ -71,6 +73,12 @@ func init() {
 		"",
 		"Run command inside a toolbox container for a different operating system release than the host")
 
+	flags.IntVarP(&runFlags.preserve_fds,
+		"preserve-fds",
+		"",
```

Comment:
... this empty parameter can be dropped.

Reply:
Done.